graphql/schema.graphql

```diff
@@ -488,6 +489,11 @@ type ClientBinary {
   displayName: String
 }
 
+type SiteBanner {
```

Comment:
Let's map this to rest/model/APIBanner in the gqlgen config. That one has the fields nullable, but they won't ever be null when reading from the db